content/2017/1/11/typography.rst

```diff
@@ -25,6 +25,10 @@ after:
 How to Choose Engaging & Accessible Typography for Your Website Brand
 =====================================================================
 
+.. callmacro:: blog/macros.j2#byline
+   get_blog_entries_by_bird('kit')[0]
```

Comment:
We had discussed handling named arguments (in the form of rST directive options); what's the reason for switching to unnamed? I suspect that it may be because docutils expects an `option_spec` mapping specific names to a converter function, which isn't general enough for our purposes here. But a cursory reading of docutils code suggests that maybe we could set `option_spec = defaultdict(lambda: str)` to read all options as strings.